src/mmw/mmw/settings/base.py

```diff
@@ -100,7 +100,7 @@ def get_env_setting(setting):
 }
 
 POSTGIS_VERSION = tuple(
-    map(int, environ.get('DJANGO_POSTGIS_VERSION', '2.1.3').split("."))
+    map(int, environ.get('DJANGO_POSTGIS_VERSION', '2.3.7').split("."))
```

Comment:
Defaulted to the exact version provided by Amazon RDS on PostgreSQL 9.6.

See: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html